scripts/items/daybreak.lua

```diff
@@ -0,0 +1,23 @@
+-----------------------------------
+-- ID: 22040
+-- Daybreak
+-- Able to cast "Dispelga"
+-----------------------------------
+---@type TItem
+local itemObject = {}
+
+itemObject.onItemCheck = function(target, item, param, caster)
+end
+
+itemObject.onItemEquip = function(target, item)
+    local mainWeapon = target:getEquippedItem(xi.slot.MAIN)
+    if mainWeapon and mainWeapon:getID() == xi.item.DAYBREAK then
+        target:addSpell(xi.magic.spell.DISPELGA, true)
```

Comment:
This will save the player's learned spells, you'll want to add another param as `false`
```suggestion
        target:addSpell(xi.magic.spell.DISPELGA, true, false)
```